### ISSUE TITLE
[FEATURE] Update default regex to accept apostrophe in local part in accordance with spec

### ISSUE THREAD

**Issue by adzap** [truemail-rb/truemail#159]
### New feature request checklist

- [x] I have updated `truemail` to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/truemail-rb/truemail/blob/master/CONTRIBUTING.md)
- [x] I have read the [documentation](https://truemail-rb.org/truemail-gem)
- [x] I have searched for [existing GitHub issues](https://github.com/truemail-rb/truemail/issues)

### Feature description
The spec https://datatracker.ietf.org/doc/html/rfc3696#page-6 allows for apostrophes in the local part of the address. Would be grateful if this was included in the default regex to avoid maintaining my own custom version.

```
>> Truemail::VERSION
 "2.4.2"
>> Truemail.configuration.email_pattern.match?("foo'bar@example.com")
 false
```

**Comment by bestwebua** [comment]
@adzap, already in latest release.

**Comment by adzap** [comment]
Oh damn sorry. Didn’t even realise I was behind.

**Comment by bestwebua** [comment]
> Oh damn sorry. Didn’t even realise I was behind.
> […](#)
> On 29 Jun 2021, 6:52 PM +1000, Vladislav Trotsenko ***@***.***>, wrote: Closed #159. — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.

@adzap, it's today's release, man ;)

**Comment by adzap** [comment]
Ah ok I gotcha. Thanks for prompt turnaround.
